mplc/doc/documentation.md

```diff
@@ -532,14 +541,14 @@ The data labels can take whatever shape you need, with only one condition. The l
 
 The `Dataset` object must provide separate train and test sets (referred to as global train set and global test set).
 
-The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set. 
+The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set.
```

Comment:
outdated ... Now the test set is fixed by the dataset, and the validation set size can be set via val_proportion
the methods do not exist anymore